src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,20 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { UserInfo } from '../UserInfo/UserInfo';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => (
+  <article className={
+    classNames('TodoInfo', { 'TodoInfo--completed': todo.completed })
```

Comment:
you need `data-id` here
<img width="711" alt="image" src="https://user-images.githubusercontent.com/55802630/184603103-8a61db6d-a0eb-4fa2-9387-224c7f8af8f2.png">